src/vconsole/vconsole-setup.c

```diff
@@ -559,6 +567,9 @@ static int find_source_vc(char **ret_path, unsigned *ret_idx) {
                 return TAKE_FD(fd);
         }
 
+        if (err == -EBUSY)
+                return log_notice_errno(err, "All existing virtual consoles are currently busy, skipping.");
```

Comment:
this is a bit unclean. if this is not considered an error, then it should not return an error code. Given you log at LOG_NOTICE level here, you don't really consider this an error (and the call site makes that clear), but then you shouldn't return an error here.

i.e. the usual way we do this is by returning either 0 or 1 depending on whether it we shall "all good, but skip" or "all good, continue". or something like that.